sources/packages/web/src/types/contracts/StudentContract.ts

```diff
@@ -3,6 +3,9 @@ import { StudentProfileAPIOutDTO } from "@/services/http/dto/Student.dto";
 export interface StudentFormInfo extends StudentProfileAPIOutDTO {
   birthDateFormatted: string;
 }
+export interface StudentFormWithRestrictionInfo extends StudentFormInfo {
```

Comment:
I would prefer to call it AEST (AESTStudentForm). We added more properties in the future keeping this name can be an issue.